src/mumble/Markdown.cpp

```diff
@@ -344,7 +344,7 @@ void escapeCharacter(QString &str, int &offset) {
 
 	tmp = tmp.toHtmlEscaped();
 
-	if (tmp.size() == 1 && tmp == str[offset]) {
+	if (tmp.size() == 1 && tmp[0] == str[offset]) {
```

Comment:
```suggestion
	if (tmp.size() == 1 && tmp.front() == str[offset]) {
```

Reply:
I think I'd prefer the more commonly known and recognized operator[] :thinking: